### DISCUSSION TITLE
Marketplace: show nr of downloads

### DISCUSSION THREAD

**Discussion by babsgosgens** [gameserverapp/Platform#990] [Suggestions]
Show number of 'downloads' (nr of times added to dashboard) to show popularity.

**Comment by mvaessen** [comment]
Thanks for posting your suggestions! Much appreciated <3

You mean like this?  :D
![Screenshot_2024-11-30_at_01_20_47](https://github.com/user-attachments/assets/5047b2fc-a8bc-4bc6-bc08-7d4fce7e4e6e)

**Comment by babsgosgens** [comment]
Hihi - indeed.

Idk how I missed that <3

**Reply by babsgosgens** [reply]
Ah - I remember why I missed that; was browsing on mobile. Perhaps you could move it closer to the download button? That's when I thought the user may need the extra encouragement to take the action.

![image](https://github.com/user-attachments/assets/d6fd5d56-c1e0-476d-8a74-c57dac6e8e0a)
![image](https://github.com/user-attachments/assets/711075df-e31c-4a67-aee1-411321f99da7)

Edit: Or perhaps underneath the title... anyway... you get it lol

**Comment by mvaessen** [comment]
Thanks for the suggestion! this should be added on the next update
